src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -693,76 +693,92 @@ public with sharing class CRLP_RollupUI_SVC {
+                jobId = CRLP_ConfigBuilder_SVC.deployedQueuedMetadataTypes();
+            }
 
             // Return the unique record DeveloperName value of the Rollup__mdt record
-            return jobId + '-' + rollupCMTObj.recordName;
+            return jobId + '-' + cmtRollup.recordName;
+
         } catch (Exception ex) {
             ERR_Handler.processError(ex, ERR_Handler_API.Context.CRLP);
             throw new AuraHandledException('ERROR: ' + ex.getMessage());
-
         }
     }
 
     /*******************************************************************************************************
      * @description Handle the save (insert/update) of a Rollup__mdt record
-     * @param rollupCMT Stringified structure of a CRLP_RollupCMT.Rollup wrapper class
+     * @param filterGroupCMT Stringified structure of a CRLP_RollupCMT.FilterGroup wrapper class
      * @return {JobId}-{DeveloperName} - ASync Job Id - Developer name of the newly inserted or updated
      * record; or an error message. The {DeveloperName} value can be used by the JS to retrieve the record after
      * the insert (deployment) completes.
      */
     @AuraEnabled
-    public static String saveFilterGroupAndRules(String filterGroupCMT) {
+    public static String saveFilterGroupAndRules(String jsonFilterGroup) {
         try {
             // For new records, the RecordName property will be null by default. The process of deploying the
             // CMT data will generate a unique RecordName to use for the new record.
-            CRLP_RollupCMT.FilterGroup filterGroupCMTObj = (CRLP_RollupCMT.FilterGroup) JSON.deserialize(filterGroupCMT, CRLP_RollupCMT.FilterGroup.class);
+            CRLP_RollupCMT.FilterGroup cmtFilterGroup = (CRLP_RollupCMT.FilterGroup) JSON.deserialize(jsonFilterGroup, CRLP_RollupCMT.FilterGroup.class);
 
-            if (filterGroupCMTObj.isDeleted == true) {
+            if (cmtFilterGroup.isDeleted == true) {
                 Map<Id, List<Rollup__mdt>> rollupsByFilterGroup = CRLP_Rollup_SEL.getRollupsByFilterGroup();
-                Integer countRollups = rollupsByFilterGroup.containsKey(filterGroupCMTObj.recordId) ? rollupsByFilterGroup.get(filterGroupCMTObj.recordId).size() : 0;
+                Integer countRollups = rollupsByFilterGroup.containsKey(cmtFilterGroup.recordId) ? rollupsByFilterGroup.get(cmtFilterGroup.recordId).size() : 0;
 
                 if (countRollups > 0) {
                     throw new RollupException(Label.CMT_FilterGroupDeleteError);
                 }
             }
 
             // Queue up the deployment
-            CRLP_ConfigBuilder_SVC.queueRollupConfigForDeploy(new List<CRLP_RollupCMT.FilterGroup>{filterGroupCMTObj});
+            CRLP_ConfigBuilder_SVC.queueRollupConfigForDeploy(new List<CRLP_RollupCMT.FilterGroup>{cmtFilterGroup});
+
+            FilterGroupUtil util = new FilterGroupUtil();
+            
+            String jobId;
+            if (util.isRollupStateEnabled()) {
```

Comment:
When a filter group is cloned and saved, the change event is not sent. The same is applicable to the filter group deletion. Can you please check if you can reproduce behaviour?

Reply:
Correct me if I'm wrong but when a filter group is cloned (a new one created) there are no referenced rollups yet - so no rollup has changed.  That's why we don't send any change event.  Also when a rollup is deleted (it can only be deleted when there are no referenced rollups) so there's no change event as well.

Reply:
Good point! Thanks for the clarification. Let's ensure we have these scenarios in a test charter indicating no change event will be created.

Reply:
Great questions. Absolutely right on both. The only changed event on a Filter that would need t be sent is when a Rule is added, deleted or modified on a Filter Group that is attached to one or more active Rollups. In that case, all the Rollup Definitions associated with that Filter Group would be considered changed.